feat: add component showcase (hold)

Blackbaud-SteveBrush opened the pull request.

Blackbaud-SteveBrush committed:
feat(components/manifest): generate manifest for code examples

Blackbaud-SteveBrush committed:
tests

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
add extra tags

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
coverage

Blackbaud-SteveBrush committed:
more tests

Blackbaud-SteveBrush committed:
100% coverage

Blackbaud-SteveBrush committed:
configure manifest project affected

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
revert

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
fix

Blackbaud-SteveBrush committed:
kinda works

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
more cleanup

Blackbaud-SteveBrush committed:
use highlight.js

Blackbaud-SteveBrush committed:
works

Blackbaud-SteveBrush committed:
works

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
styles

Blackbaud-SteveBrush committed:
fix tests

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
use sky-prefix

Blackbaud-SteveBrush committed:
add title

Blackbaud-SteveBrush committed:
try fix build

blackbaud-sky-build-user commented:
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/3142/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/a11y-storybook/)
- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/colorpicker-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/core-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/forms-storybook/)
- [help-inline-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/help-inline-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/3142/storybooks/toast-storybook/)

Apps:

- [code-examples-playground](https://blackbaud.github.io/skyux-pr-preview/3142/code-examples-playground/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/3142/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/3142/playground/)

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
copy to clipboard

Blackbaud-SteveBrush committed:
basic stuff

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
more cleanup

Blackbaud-SteveBrush committed:
cleanup style

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
create packages

Blackbaud-SteveBrush committed:
tests

Blackbaud-SteveBrush committed:
more tests

Blackbaud-SteveBrush committed:
more tests

Blackbaud-SteveBrush committed:
more tests

Blackbaud-SteveBrush committed:
more tests

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
updates

Blackbaud-SteveBrush committed:
fix tests

Blackbaud-SteveBrush committed:
fix build

Blackbaud-SteveBrush committed:
fix build

Blackbaud-SteveBrush committed:
make prebuild cacheable

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
tests

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
pill

Blackbaud-SteveBrush committed:
console.logs

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
fix manifest

Blackbaud-SteveBrush committed:
fix manifest

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
updates

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
style cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
fix routing

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
different approach

Blackbaud-SteveBrush committed:
code style cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
fixes

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
tests

Blackbaud-SteveBrush committed:
create code highlight service/pipe

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
fix build

Blackbaud-SteveBrush committed:
add example titles

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
inputs/outputs

Blackbaud-SteveBrush committed:
font size

Blackbaud-SteveBrush committed:
small tweaks

Blackbaud-SteveBrush committed:
style stuff

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
styles

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
style cleanup

Blackbaud-SteveBrush committed:
finalize pill component

Blackbaud-SteveBrush committed:
cleanup clipboard

Blackbaud-SteveBrush committed:
cleanup clipboard

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
remove flourishes

Blackbaud-SteveBrush committed:
more cleanup

Blackbaud-SteveBrush committed:
box cleanup

Blackbaud-SteveBrush committed:
toc cleanup

Blackbaud-SteveBrush committed:
more cleanup

Blackbaud-SteveBrush committed:
remove br tags

Blackbaud-SteveBrush committed:
fix paragraphs in jsdocs

Blackbaud-SteveBrush committed:
use md

Blackbaud-SteveBrush committed:
new components

Blackbaud-SteveBrush committed:
updates

Blackbaud-SteveBrush committed:
updates

Blackbaud-SteveBrush committed:
installation module

Blackbaud-SteveBrush committed:
use box

Blackbaud-SteveBrush committed:
npm install command

Blackbaud-SteveBrush committed:
fix styles

Blackbaud-SteveBrush committed:
coverage

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
add git repo url

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
works

Blackbaud-SteveBrush committed:
fix toc ordering

Blackbaud-SteveBrush committed:
more cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
move type-definition to docs-tools

Blackbaud-SteveBrush committed:
cleanup showcase

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
more components

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
manifest tests

Blackbaud-SteveBrush committed:
more coverage

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
remove anatomy components

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
feat(components/manifest): separate class properties and methods

separate class properties and methods; validate docsIds; add package information; fix output for
property accessors; add repo URLs to each type

Blackbaud-SteveBrush committed:
Merge branch 'main' into manifest-fixes

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/manifest-fixes' into manifest-project-config

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
package

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into docs-tools-2

Blackbaud-SteveBrush committed:
add docs tools

Blackbaud-SteveBrush committed:
minor adjustments

Blackbaud-SteveBrush committed:
fix lint

Blackbaud-SteveBrush committed:
fix build

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/docs-tools-2' into manifest-project-config

Blackbaud-SteveBrush committed:
more titles

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
remove path

Blackbaud-SteveBrush committed:
add more example titles

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config

Blackbaud-SteveBrush committed:
fix code-examples

Blackbaud-SteveBrush committed:
cleanup

Blackbaud-SteveBrush committed:
fixes

Blackbaud-SteveBrush committed:
Merge remote-tracking branch 'origin/main' into manifest-project-config
